language/move-compiler/tests/move_check/parser/invalid_character_comment.exp

```diff
@@ -1,6 +1,6 @@
-error[E01001]: invalid character
+error[E01001]:invalid character
   ┌─ tests/move_check/parser/invalid_character_comment.move:3:44
   │
 3 │ // Non-ASCII characters in comments (e.g., ф) are also not allowed.
-  │                                            ^ Invalid character 'ф' found when reading file. Only ASCII printable characters, tabs (\t), and line endings (\n) are permitted.
+  │                                            ^ Invalid character '\u{444}' found when reading file. Only ASCII printable characters, tabs (\t), and line endings (\n) are permitted.
```

Comment:
I now have a much better understanding of whats going on here, thanks for the test.

Would it be possible to display the original character if it is printable? Not sure if there is an easy way to check that or not.

If not, this at least fixes the bug, so we can land as is